analyzer_test.go

```diff
@@ -75,3 +75,39 @@ func TestArgsCorrect(t *testing.T) {
 		},
 	})))
 }
+
+var pathTests = []struct {
+	in  string
+	out string
+}{
+	{"a/b.go", "/tmp/a___.___b.go"},
+	{"tmp/a/b.go", "/tmp/tmp___.___a___.___b.go"},
+	{"a/b/c/d/e.go", "/tmp/a___.___b___.___c___.___d___.___e.go"},
+}
+
+func TestPathTransformations(t *testing.T) {
+	for _, tt := range pathTests {
+		t.Run(tt.in, func(t *testing.T) {
+			flat := faltternPath(tt.in, "/tmp")
+			if flat != tt.out {
+				t.Errorf("forward: got %q, want %q", flat, tt.out)
```

Comment:
why not `assert.Equal(t, tt.out, faltternPath(tt.in, "/tmp"))`?

Reply:
Addressed in 425f928